flowkit/gateway/emulator.go

```diff
@@ -148,6 +148,11 @@ func (g *EmulatorGateway) GetTransactionResultsByBlockID(_ flow.Identifier) ([]*
 	panic("GetTransactionResultsByBlockID not implemented")
 }
 
+func (g *EmulatorGateway) GetTransactionResultByIndex(_ flow.Identifier, _ uint32) (*flow.TransactionResult, error) {
+	// TODO: implement
```

Comment:
Should this be implemented before merge?

Reply:
There are plenty of other operations in EmulatorGateway that is not implemented. I am not even sure how to do it.